terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/codelist/01_01_form.jsp

```diff
@@ -41,8 +41,6 @@
       </tr>
```

Comment:
- [ ] There seems to be some inconsistencies in addition to what we have corrected this time.

Reply:
d1b76ae2ebbc888216493ece810f760b8bcedd51
The omitted correction of the deletion of duplicate tags has been addressed.